views/event/create.ejs

```diff
@@ -3,27 +3,54 @@
 <head>
     <% include ../template/head.ejs %>
     <script>
+        // default date
+        window.onload= function() {
+            let now = new Date();
+            let day = ("0" + now.getDate()).slice(-2);
+            let month = ("0" + (now.getMonth() + 1)).slice(-2);
+            let today = now.getFullYear()+"-"+(month)+"-"+(day) ;
+            $("#time").val(today);
+        }
+        // validate URL
+        function validURL(str) {
+            var pattern = new RegExp('^(https?:\\/\\/)?'+ // protocol
+                '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|'+ // domain name
+                '((\\d{1,3}\\.){3}\\d{1,3}))'+ // OR ip (v4) address
+                '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*'+ // port and path
+                '(\\?[;&a-z\\d%_.~+=-]*)?'+ // query string
+                '(\\#[-a-z\\d_]*)?$','i'); // fragment locator
+            return !!pattern.test(str);
+        }
```

Comment:
Could you move the validate url and add https to a new js file for future reuse? Thanks

Reply:
the file can be called something like format.js

Reply:
I found a file in public/js/ called "formatchecker".  I put the validURL() function inside this file since it also contains other functions for validation.